### PR TITLE
Add event description

### PR DESCRIPTION
**Pull request by gardenofwine**

### CODE REVIEW

**Comment by avihut** [comment]
Looks good :+1:

**Comment by ShayDavidson** [comment]
Can you add an img?

File: app/src/main/java/com/paypal/psix/adapters/EventsAdapter.java [hunk 1]
```diff
@@ -64,6 +64,7 @@ public View getView(int position, View view, ViewGroup parent) {
 
         holder.titleLabel.setText(event.name);
         holder.dateLabel.setText(event.getShortFormattedDate());
+        holder.descLabel.setText(event.description.substring(0, Math.min(event.description.length(), 25)));
```
Comment (ShayDavidson) [inline app/src/main/java/com/paypal/psix/adapters/EventsAdapter.java:67]: You can set an ellipsis on the text view, don't truncate it manially

**Comment by gardenofwine** [comment]
@ShayDavidson What image do you mean? The event list already displays the event's main image

**Pull request merged**